pie_docker.py

```diff
@@ -0,0 +1,27 @@
+"""
+Python3.6+ only
+"""
+from pie import *
+
+
+
+class Docker:
+    def __init__(self,options=[]):
+        self.options=options
+
+    def build(self,context,options=[]):
```

Comment:
which is probably not a big deal because it's never called twice, but anyway

Reply:
Well, it's read only... unless it doesn't become so, I guess :-)

Reply:
(will fix, thank you)